.github/workflows/docker.yml

```
@@ -6,8 +6,8 @@ on:
       - rc*/*
   push:
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
-      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v20.15.10-rc5, v20.15.10-rc12+build1
```

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+-rc.*" # Push events to matching v*, i.e. v20.15.10-rc5, v20.15.10-rc12+build1
```
Don't we need `.` before `*` in order to include the words(5, 12+build1) right after `rc`?
When I try this in online regex site(https://regex101.com/), This doesn't seem to match to `v20.15.10-rc5, v20.15.10-rc12+build1`
Could you check it?

---

This is not regex and this pattern is according to https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

---

Okay. Thank you for the link.